package.json

```diff
@@ -25,13 +25,6 @@
     "docs:test": "pnpm --filter design-system-documentation test",
     "docs:e2e": "start-server-and-test docs:headless 9300 'pnpm --filter design-system-documentation e2e'",
     "docs:e2e:watch": "start-server-and-test docs:headless 9300 'pnpm --filter design-system-documentation e2e:watch'",
-    "docsv6": "pnpm --filter design-system-documentation-v6 start",
-    "docsv6:start": "pnpm --filter design-system-documentation-v6... --parallel --stream start",
-    "docsv6:build": "pnpm --filter design-system-documentation-v6 build",
-    "docsv6:headless": "pnpm --filter design-system-documentation-v6 start:headless",
-    "docsv6:e2e": "start-server-and-test docsv6:headless 9300 'pnpm --filter design-system-documentation-v6 e2e'",
-    "docsv6:e2e:watch": "start-server-and-test docsv6:headless 9300 'pnpm --filter design-system-documentation-v6 e2e:watch'",
-    "docsv6:snapshots": "start-server-and-test 'pnpm docsv6:headless' 9300 'pnpm --filter design-system-documentation-v6 snapshots'",
```

Comment:
Don't we need the snapshot script for the current docs package?

Reply:
There's still the "snapshots" command that directly invokes the script in the documentation/package.json.